bundles/core/org.eclipse.smarthome.core.test/src/test/java/org/eclipse/smarthome/core/net/NetUtilTest.java

```diff
@@ -29,45 +30,79 @@
     public void testNetwork() {
         String network = NetUtil.getIpv4NetAddress("192.168.0.1", (short) 24);
         assertThat(network, is("192.168.0.0"));
+        String broadcast = NetUtil.getIpv4NetBroadcastAddress("192.168.0.1", (short) 24);
```

Comment:
Thank you for creating these test cases!

Since they test something different than the ip address, I guess it would be better to separate them in an extra method.

Please move your lines into a new method:
```
@Test
public voidTestBroadcastAddress() {
//your code
}
```

The `@Test` annotation tells junit that it should treat this method as a test case and execute it.

Reply:
Done.